app/Resources/views/security/adherent_login.html.twig

```diff
@@ -0,0 +1,44 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}
+    {{ parent() }} | Connexion
+{% endblock %}
+
+{% block content %}
+    <main>
+        <section class="l__wrapper">
+            <div class="login l__col background--light space--20-30 l__tablet--col b__nudge--bottom-large">
+                <h2 class="text--center">Accédez à votre espace adhérent</h2>
+                <div class="login--box l__col space--20-30">
+                    <p class="text--center">Connectez-vous à votre espace adhérent</p>
+
+                    {{ form_start(form, { class: 'l__row l__row--wrap' }) }}
+                        {{ form_widget(form._adherent_email, { attr: {
+                            placeholder: 'Votre e-mail',
+                            'aria-label': 'Votre e-mail',
+                            class: 'form__field form__row form--full'
+                        } }) }}
+                        {{ form_widget(form._adherent_password, { attr: {
+                            placeholder: 'Mot de passe',
+                            'aria-label': 'Mot de passe',
+                            class: 'form__field form__row form--full'
```

Comment:
The form theme should already add the class `form__field` (should be addressed in all spots).